exporter/trace/otelcol/README.md

```diff
@@ -0,0 +1,24 @@
+# OpenTelemetry Collector Go Exporter
+
+[![GoDoc][godoc-image]][godoc-url]
+
+
+This repository contains the Go implementation of the OpenTelemetry Collector Exporter.
+OpenTelemetry Agent is a deamon process running in a VM that can retrieve spans/stats/metrics from
```

Comment:
This sentence was probably copied from OC agent doc. Needs to be rephrased or just replaced with a link to https://github.com/open-telemetry/opentelemetry-collector/.

There is no OpenTelemetry Agent, but a Collector which can be deployed as agent/daemon/sidecar.
Also, this sentence doesn't mention that collector could also be running independently, not as a daemon.
And why mention VM? It can probably run on bare metal? Is that even worth mentioning here?

Reply:
Removed the original text replaced with simple one line for now.